### PR TITLE
Issue 1 creating an entry

### PR DESCRIPTION
**Pull request by dcatan89**
Screen Shot 2021-12-22 at 3 42 44 PM
Screen Shot 2021-12-22 at 3 37 38 PM
Screen Shot 2021-12-22 at 3 51 41 PM
Screen Shot 2021-12-22 at 3 52 02 PM

### CODE REVIEW

**Comment by dcatan89** [comment]
Screen Shot 2021-12-22 at 4 30 17 PM

File: index.html [hunk 1]
```diff
@@ -1,25 +1,62 @@
...
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
-<body>
-  <header>
-  </header>
-  <main>
-  </main>
+<body class="bground-off-white">
+  <div class="container background-blviolet">
```
Comment (codingcodymiller) [inline index.html]: You should put the background color on the `header` element, and make that element have a child that is a `div` with the `container` class, along with the row and column you currently have surrounding the `header` element. The `header` element is meant to be the parent for all the header content.

File: index.html [hunk 2]
```diff
@@ -1,25 +1,62 @@
...
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-half placehld-img">
+          <img src="images/placeholder-image-square.jpg">
+        </div>
+        <div class="column-half">
+          <div class="row title-section">
+            <label for="user-title">Title</label>
+            <input required name="title" type="text" id="title-entry">
```
Comment (codingcodymiller) [inline index.html]: These `for` and `id` attributes don't match

File: index.html [hunk 3]
```diff
@@ -1,25 +1,62 @@
...
+          </div>
+          <div class="row url-section">
+            <label for="user-url">Photo URL</label>
+            <input required name="url" type="url" id="url-entry">
```
Comment (codingcodymiller) [inline index.html]: These `for` and `id` attributes don't match

File: index.html [hunk 4]
```diff
@@ -1,25 +1,62 @@
...
+          </div>
+
+        </div>
+      </div>
+      <div class="notes-section row">
+        <label for="user-notes">Notes</label>
+        <textarea required rows="8" name="notes" id="notes-entry"></textarea>
```
Comment (codingcodymiller) [inline index.html]: These `for` and `id` attributes don't match

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by dcatan89** [comment]
Screen Shot 2021-12-22 at 4 49 57 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Your `for` attributes on your `label` elements still do not match the `id` attribute on their corresponding form control element.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**